resources/assets/components/InboxItem/index.js

```diff
@@ -4,20 +4,22 @@ import { remove, map, clone } from 'lodash';
 
 import Tags from '../Tags';
 import InboxTile from './InboxTile';
+import StatusButton from '../StatusButton';
```

Comment:
Think this branch needs to be rebased? not sure why the status button stuff is showing up in this PR 🤔